apps/system/js/activities.js

```diff
@@ -12,6 +12,7 @@
   function Activities() {
     window.addEventListener('mozChromeEvent', this);
     window.addEventListener('appopened', this);
+    navigator.mozApps.mgmt.addEventListener('install', this);
```

Comment:
missed it during the first review, sorry.
we generally try to keep all `mozApps.mgmt` listeners in `applications.js` in the sytem app.

you should listen to `applicationinstall` on the window here instead (dipatched by `applications.js`)